catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundBrick.java

```diff
@@ -32,7 +32,7 @@ public SetBackgroundBrick() {
 	}
 
 	@Override
-	protected Sprite getSprite() {
+	protected Sprite getSpriteToChange() {
```

Comment:
Please revert this change. It's not important for the current release. I'll explain to you what I meant at https://github.com/Catrobat/Catroid/pull/1923#discussion_r83840129 later. :wink: